modules/telestion-api/src/main/java/de/wuespace/telestion/api/verticle/GenericConfiguration.java

```diff
@@ -1,4 +1,10 @@
 package de.wuespace.telestion.api.verticle;
 
+/**
+ * The Configuration class which should be used if no config is needed to infer the generics of a
+ * {@link TelestionVerticle}.
+ *
+ * @author Cedric Boes (cb0s), Ludwig Richter (@fussel178)
+ */
 public record GenericConfiguration() implements TelestionConfiguration {
```

Comment:
Maybe better to name it `NoConfig`, `EmptyConfig` or `BaseConfig` something along the lines. But generic makes me think of generics in this context.